docker/centos/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+set -e
+set -x
```

Comment:
I hope this should be skipped in normal builds

Reply:
Removed